fandogh_cli/base_commands.py

```diff
@@ -78,8 +78,9 @@ def _get_latest_version(self):
     def _check_for_error_collection_permission(self):
         collect_error = get_user_config().get('collect_error')
         if collect_error is None:
-            if os.environ.get('COLLECT_ERROR', False):
-                get_user_config().set('collect_error', 'YES')
+            env_value = os.environ.get('COLLECT_ERROR', None)
+            if env_value in ['YES', 'NO']:
```

Comment:
Thank you for your contribution. 🙏 
This change is a bit sensitive since many users are using fandogh CLI in their automated scripts, changing this logic might break their automation since fandogh might ask for input during an automated script.
If there was an environment variable named COLLECT_ERROR, even if it wasn't one of the YES / NO values, we shouln't ask for user input.
as a suggestion, I think we can take all the values like `('0', 'n', 'No', 'no', 'NO')` as NO and `(1, 'y', 'yes','Yes', 'YES' )` as YES, and any other value (except None) as NO.

Reply:
Hi, @MahdiZareie, you're right

```python
if os.environ.get('COLLECT_ERROR', False):                		 
  	get_user_config().set('collect_error', 'YES')
```

> and any other value (except None) as NO.

I think it should be `YES`.